x-pack/plugins/case/server/routes/api/cases/configure/get_fields.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Boom from '@hapi/boom';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { fold } from 'fp-ts/lib/Either';
+import { identity } from 'fp-ts/lib/function';
+import { RouteDeps } from '../../types';
+import { escapeHatch, wrapError } from '../../utils';
+
+import { CASE_CONFIGURE_CONNECTOR_DETAILS_URL } from '../../../../../common/constants';
+import {
+  ConnectorRequestParamsRt,
+  GetFieldsRequestQueryRt,
+  throwErrors,
+} from '../../../../../common/api';
+
+export function initCaseConfigureGetFields({ router }: RouteDeps) {
+  router.get(
+    {
+      path: CASE_CONFIGURE_CONNECTOR_DETAILS_URL,
```

Comment:
`CASE_CONFIGURE_CONNECTOR_DETAILS_URL` is defined as `api/cases/configure/connectors/{connector_id}`. I think it is confusing to return the fields from this route. This route usually should return everything about the specific connector such as details from the Action API, mapping etc.

In my opinion, a better naming would be `api/cases/configure/connectors/{connector_id}/fields`.